site/docs/v1/tech/lambdas/client-credentials-jwt-populate.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: doc
+title: Client Credentials JWT Populate lambda
+description: An overview of the Client Credentials JWT Populate lambda
+---
+
+:sectnumlevels: 0
+
+== Client Credentials JWT Populate lambda
+
+If you would like to augment the claims provided in the JWT before it has been signed you can specify a lambda in the JWT configuration. This lambda will be invoked prior to the token being signed and issued to a user.
```

Comment:
In the client credentials grant, there is not a user. The token will be issued as a result of the client credentials grant, on behalf of an entity.

Reply:
Ah, yes.  Makes sense.

Reply:
> This lambda will be invoked prior to the token being signed and issued to a user.

What I mean by this comment was that we should not reference a "user", as there is no user in this context. Make sense?

Reply:
Yeah, now it does.  I removed the reference to a user.  Unless we want to call it by one of these names :) 
https://en.wikipedia.org/wiki/List_of_fictional_computers